git-to-freshports/git-to-freshports-xml.py

```diff
@@ -89,7 +94,15 @@ def main():
     log.debug(f"Config is: {config}")
 
     repo = git.Repo(str(config['path']))
-    commits = reversed(list(repo.iter_commits(f"{config['commit']}..HEAD")))
+
+    if config['commit']:
+        commits = reversed(list(repo.iter_commits(f"{config['commit']}..HEAD")))
+    elif config['commit_range']:
+        commits = reversed(list(repo.iter_commits(config['commit_range'])))
+    elif config['single_commit']:
+        commits = [repo.commit(config['single_commit'])]
```

Comment:
This!  This is what I didn't know how to do. It's easy when I see how. Thank you.